_data/glossary/kube-proxy.yaml

```diff
@@ -0,0 +1,10 @@
+id: kube proxy
+name: Kube-proxy
+full-link: /docs/reference/generated/kube-proxy
+tags:
+- fundamental
+- core-object
+short-description: >
+  Kube proxy is a network proxy that runs on each node in the cluster. Kube proxy is responsible for request forwarding.
+long-description: >
+  The kube proxy allows TCP and UDP stream forwarding or round robin TCP and UDP forwarding across a set of backend services.
```

Comment:
Please change "kube proxy" back to "kube-proxy".
Also be careful about the term "services" here. In Kubernetes, `Service` is a special term.

Reply:
Replace with:
> `kube-proxy` is responsible for request forwarding. `kube-proxy` allows TCP and UDP stream forwarding or round robin TCP and UDP forwarding across a set of backend functions.

"Functions" is clunky, but like @tengqm says, we have to be careful with "Services".